docs/components/modeler/desktop-modeler/process-applications.md

```diff
@@ -12,13 +12,19 @@ import StartInstanceImg from './img/process-applications/start-instance.png'
 
 Desktop modeler recognizes [process applications](../../concepts/process-applications.md) you build and offers you advanced editor intelligence, deployment, and execution features within the context of such an application. To identify the boundaries of a process application, Desktop Modeler searches for a `.process-application` file in the root of your project.
 
-For instance, a consumer loan approval process application might contain:
+In professional software development, a typical process application contains resources such as BPMN, DMN, and Form files. These live alonside [job workers](/components/concepts/job-workers.md), implementing process logic, additional application code, and tests. How exactly your project is structured may vary depending on the implementation language, libraries, and frameworks you use.
+
+## Example: Consumer loan application
+
+Consider an application implementing consumer loan approval. It may contain:
 
 - A main BPMN process (for example, `consumer-loan-application.bpmn`) to define the workflow.
 - DMN decisions (for example, `interest-rate-calculation.dmn`, `credit-score-calculation.dmn`) for business rules.
 - Forms (for example, `loan-application-review.form`) for user interactions.
+- Various [job workers](/components/concepts/job-workers.md) that implement process behavior.
+- Additional application code and tests
 
-The structure of your project may vary depending on the implementation language. For example, a standard Java/Maven might be structured as follows:
+In a standard standard Java/Maven project, such an application may be structured as follows:
```

Comment:
[❓] What do you think about the following wording? If not, we should remove the second `standard`
```suggestion
In a typical Java/Maven project, the structure of such an application might be as follows:
```